### ISSUE TITLE
Enable all cops disabled by default

### ISSUE THREAD

**Issue by yegor256** [rubocop/rubocop#15088]
Would be nice to have an ability to enable _all_ cops that have `Enabled` set to `false`.  Maybe like this:

```
AllCops:
  Enabled: true
```

This will allow me to see the full scope of Rubocop analysis and then disable what's not a good fit for my project. Currently, I can't do this and have to go through the documentation and enable one by one.

**Comment by koic** [comment]
It could be achieved using the existing `EnabledByDefault: true` feature.

```yaml
AllCops:
  EnabledByDefault: true
```

https://docs.rubocop.org/rubocop/latest/configuration/cop_settings.html
